### ISSUE TITLE
Rename /json to something more descriptive like /status

### ISSUE THREAD

**Issue by marvinmarnold** [NebraLtd/hm-diag#247]

**Comment by shawaj** [comment]
As part of things like this, and further to something @kashifpk said somewhere (can't find it right now or would link to it) we should try to document where endpoints are used and ideally add testing on both sides.

Right now a change like this requires knowledge of where /json is used in order to update it everywhere... If we had better documentation and testing we could make our flow much more productive and less reliant on individuals knowing the codebase really well to understand where things are used (which will become impractical/ impossible in the future as the codebase grows anyway

**Comment by kashifpk** [comment]
Yes if we have integration tests that rely on actual responses for components (on testnet) then we could quickly find out places our change broke and fix. Even for individuals who know about all the places where such endpoints are used it would be troublesome to keep track of them.

Adding unit tests and integration tests might feel like a pain initially but this should really help us going in circles where new changes break things and we end up spending time on fixing those any way. It's the time we spend now to make our development and testing process more robust that could save us from a lot more pain in the future.

**Comment by shawaj** [comment]
100% agree 👍
